src/control/ros/Conversions.cpp

```diff
@@ -311,6 +344,23 @@ std::unique_ptr<SplineTrajectory> toSplineJointTrajectory(
 
     rosJointToMetaSkeletonJoint.emplace(
         std::make_pair(trajIndex, metaSkeletonIndex));
+    specifiedMetaSkeletonJoints.insert(metaSkeletonIndex);
+  }
+
+  // Add unspecified joint mappings to rosJointToMetaSkeletonJoint
+  std::vector<size_t> unspecifiedMetaSkeletonJoints;
```

Comment:
Nitpick: We could reserve the size of `unspecifiedMetaSkeletonJoints` to `numControlledJoints - numTrajectoryJoints`.